### DISCUSSION TITLE
Coding on vehicle routing problem

### DISCUSSION THREAD

**Discussion by tejinder03** [TimefoldAI/timefold-solver#164] [General]
I am working on vehicle problem how to solve it  using time fold

**Comment by ge0ffrey** [comment]
We have 2 quickstarts:
- https://github.com/TimefoldAI/timefold-quickstarts/tree/stable/use-cases/vehicle-routing-capacity
- https://github.com/TimefoldAI/timefold-quickstarts/tree/stable/use-cases/vehicle-routing-time-windows

**Reply by jandalpetr** [reply]
Hello,
Non of the repo exist.
Thank you.

**Reply by triceo** [reply]
The quickstarts were recently merged into one:
https://github.com/TimefoldAI/timefold-quickstarts/tree/stable/use-cases/vehicle-routing
